packages/mom5/package.py

```diff
@@ -490,7 +487,7 @@ def build(self, spec, prefix):
             build = Executable("./MOM_compile.csh")
             if "+restart_repro" in self.spec:
                 build.add_default_env("REPRO", "true")
-            if "@access-esm1.5" not in self.spec:
+            if self.spec.satisfies("@access-esm1.5"):
```

Comment:
Should this be
```
if not self.spec.satisfies("@access-esm1.5:access-esm1.6"):
```
because of the `not` on line 493 of the previous?

Reply:
Whoops, good catch. I was ~not~ paying enough attention when I changed the syntax. I think it should be

```
if not self.spec.satisfies("@access-esm1.5")
```

because the `master` and `access-esm1.6` versions both use the master branch and should enter here.

I'm away from my computer at the mo, but will change this when I'm back.